packages/react/src/components/Header/HeaderAction/HeaderAction.test.jsx

```diff
@@ -47,4 +47,23 @@ describe('HeaderAction', () => {
     expect(screen.queryByText('myButton')).not.toBeInTheDocument();
     expect(screen.getByRole('button', { name: /myhelpLabel/i })).toBeVisible();
   });
+  it('it renders an a instead of button if item has href', () => {
+    const mockProps = {
+      testId: 'my-header-action',
+      item: {
+        label: 'myhelpLabel',
+        btnContent: <span>myButton</span>,
+        href: 'https://www.ibm.com',
+        rel: 'noopener noreferrer',
+        target: '_blank',
+      },
+      index: 0,
+      renderLabel: true,
+    };
+
+    render(<HeaderAction {...mockProps} />);
+    expect(true).toBeTruthy();
+    expect(screen.queryByText('myButton')).not.toBeInTheDocument();
+    expect(screen.getByTestId('menu-item-myhelpLabel-global')).toBeVisible();
+  });
```

Comment:
@blechner Can we make this test a little more tangible? Maybe something like: 

```suggestion
    expect(screen.getByTestId('menu-item-myhelpLabel-global')).toBeVisible();
    expect(screen.getByTestId('menu-item-myhelpLabel-global').tagName).toEqual('A');
    expect(screen.getByTestId('menu-item-myhelpLabel-global')).toHaveAttribute('href', 'https://www.ibm.com');
  });
```

Reply:
Sure, I hadn't added it before because I was not sure how to test if an html element was an `<a>`, I didn't know about this `.tagName` property of `getByTestId`. All the examples of checking if an element was `button` used the `getByRole`, and for some reason the `<a>` don't have the role attribute set, so I didn't know how to test :)
Thanks for the suggestions, I'll commit your suggestion.